README.md

```diff
@@ -0,0 +1,45 @@
+# CI Operator
+
+CI Operator automates and simplifies the process of building and testing
+OpenShift component images (i.e. any `openshift/origin-{component}` images).
+
+Given a Git repository reference and a component-specific JSON configuration
+file, describing base images and which images should be built and tested and
+how, CI Operator builds the component images within an OpenShift cluster and
+runs the tests. All artifacts are built in a new namespace named using a hash of
+all inputs, so the artifacts can be reused when the inputs are identical.
+
+CI Operator is mainly intended to be run inside a pod in a cluster, triggered by
+the Prow CI infrastructure, but it is also possible to run it as a CLI tool on a
+developer laptop.
+
+Note: CI Operator is not a [Kubernetes operator](https://coreos.com/operators/).
```

Comment:
I think this reads wierd - you might want to say "ci-operator orchestrates builds and tests, but should not be confused with [Kubernetes operators](....) which make managing software on top of Kubernetes easier."